app/grandchallenge/algorithms/templates/algorithms/algorithm_detail.html

```diff
@@ -112,6 +112,21 @@
 
 {% block content %}
     {% if 'change_algorithm' in algorithm_perms %}
+        {% if object.job_requires_gpu_type == gpu_type_choices.A10G and object.job_requires_memory_gb == 32 %}
+            <div class="alert alert-danger" role="alert">
+                <p><i class="fas fa-exclamation-triangle mr-1"></i><u>A10G GPU Availability Issue</u></p>
```

Comment:
Don't know if there is better wording? It is only an issue for high memory instances.

```suggestion
                <p><i class="fas fa-exclamation-triangle mr-1"></i><u>Instance Availability Issue</u></p>
```